frontend/test/__runner__/backend.js

```diff
@@ -41,39 +41,58 @@ const BackendResource = createSharedResource("BackendResource", {
         fs.copyFileSync(`${server.dbKey}.mv.db`, `${server.dbFile}.mv.db`);
       }
 
+      const javaFlags = [
+        "-XX:+IgnoreUnrecognizedVMOptions", // ignore options not recognized by this Java version (e.g. Java 8 should ignore Java 9 options)
+        "-Dh2.bindAddress=localhost", // fix H2 randomly not working (?)
+        "-Djava.awt.headless=true", // when running on macOS prevent little Java icon from popping up in Dock
+        "-Duser.timezone=US/Pacific",
+        `-Dlog4j.configurationFile=file:${__dirname}/log4j2.xml`,
+      ];
+
+      const databaseConfig = {
+        MB_DB_TYPE: "h2",
+        MB_DB_FILE: server.dbFile,
+        MB_JETTY_HOST: "0.0.0.0",
+        MB_JETTY_PORT: server.port,
+        MB_ENABLE_TEST_ENDPOINTS: "true",
+        MB_PREMIUM_EMBEDDING_TOKEN:
+          (process.env["MB_EDITION"] === "ee" &&
+            process.env["ENTERPRISE_TOKEN"]) ||
+          undefined,
+      };
```

Comment:
Not sure all of these settings are related to databases (as they're in `databaseConfig`). Meaning:

- `MB_JETTY_HOST`
- `MB_JETTY_PORT`
- `MB_ENABLE_TEST_ENDPOINTS`
- `MB_PREMIUM_EMBEDDING_TOKEN`

Reply:
Would `connectionDefaults` be a better name?
Any suggestions (and alternative approaches) are more than welcome :)